harness/determined/keras/callbacks.py

```diff
@@ -644,3 +647,54 @@ def get_state(self) -> Any:
 
     def load_state(self, state: Any) -> None:
         _tf_keras_callback_load_state(self, state)
+
+
+class TensorBoard(tf.keras.callbacks.TensorBoard, Callback):  # type: ignore
```

Comment:
non-blocking: can we rename this to TensorboardCallback

Reply:
No, because we're mirroring what it is called in Keras, as we do with the other names in this file.